en/power_module/REAME.md

```diff
@@ -0,0 +1,16 @@
+# Power module
```

Comment:
Please change the file name to README.md. Also change in SUMMARY.md for the link.

Reply:
@cuhome ^^^ The filename is a typo REAME not README. 

Heading title should be `# Power Modules`

Reply:
@cuhome Also see here.